### DISCUSSION TITLE
Show bpm in the mod customisation menu

### DISCUSSION THREAD

**Discussion by Maarlink** [ppy/osu#19190] [Ideas] [answered]
![osu_2022-07-17_14-39-58](https://user-images.githubusercontent.com/109466041/179398748-24381d32-9bc2-4312-a513-ed4144e686ae.jpg)

It would be cool seeing the new bpm and the original bpm of the map when you have DT/NC/HT/DC on, in the mod customisation menu, so you can speed up or slow down a map based on the bpm you want to play. Maybe placing this where is my cursor on the screenshot above.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/18189
